netstandard/ref/System.Diagnostics.Tracing.cs

```diff
@@ -51,12 +51,13 @@ public partial class EventCommandEventArgs : System.EventArgs
         public bool EnableEvent(int eventId) { throw null; }
     }
     [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)]
-    public class EventCounter
```

Comment:
Hmm some of this diff looks like someone manually added these.. not a big deal just odd.